packages/mask/src/components/shared/ApplicationSettingPluginList.tsx

```diff
@@ -0,0 +1,175 @@
+    },
+}))
+
+export function ApplicationSettingPluginList() {
+    const { classes } = useStyles()
+    const { t } = useI18N()
+    const snsAdaptorPlugins = useActivatedPluginsSNSAdaptor('any')
+    const applicationList = useMemo(
+        () =>
+            snsAdaptorPlugins
+                .reduce<Application[]>((acc, cur) => {
+                    if (!cur.ApplicationEntries) return acc
+                    return acc.concat(
+                        cur.ApplicationEntries.filter((x) => x.appBoardSortingDefaultPriority).map((x) => {
+                            return {
+                                entry: x,
+                                pluginId: cur.ID,
+                            }
+                        }) ?? EMPTY_LIST,
+                    )
+                }, EMPTY_LIST)
+                .sort(
+                    (a, b) =>
+                        (a.entry.appBoardSortingDefaultPriority ?? 0) - (b.entry.appBoardSortingDefaultPriority ?? 0),
+                ),
+        [snsAdaptorPlugins],
+    )
+    const [listedAppList, setListedAppList] = useState(applicationList.filter((x) => !getUnlistedApp(x)))
+    const [unlistedAppList, setUnListedAppList] = useState(applicationList.filter((x) => getUnlistedApp(x)))
+
+    const setAppList = useCallback(
+        (app: Application, unlisted: boolean) => {
+            setUnlistedApp(app, unlisted)
+            if (unlisted) {
+                setUnListedAppList(unlistedAppList.concat(app))
+                setListedAppList(
+                    listedAppList.filter((x) => x.entry.ApplicationEntryID !== app.entry.ApplicationEntryID),
+                )
+            } else {
+                setListedAppList(listedAppList.concat(app))
+                setUnListedAppList(
+                    unlistedAppList.filter((x) => x.entry.ApplicationEntryID !== app.entry.ApplicationEntryID),
+                )
+            }
+        },
+        [applicationList, listedAppList, unlistedAppList],
+    )
+
+    return (
+        <div>
+            <AppList appList={listedAppList} setUnlistedApp={setAppList} isListed />
+            <Typography className={classes.unlisted}>{t('application_settings_tab_plug_app-list-unlisted')}</Typography>
+            <AppList appList={unlistedAppList} setUnlistedApp={setAppList} isListed={false} />
+        </div>
+    )
+}
+
+interface AppListProps {
+    appList: Application[]
+    setUnlistedApp: (app: Application, unlisted: boolean) => void
+    isListed: boolean
+}
+
+function AppList(props: AppListProps) {
+    const { appList, setUnlistedApp, isListed } = props
+    const { classes } = useStyles()
+    const { t } = useI18N()
+
+    return appList.length > 0 ? (
+        <List className={classes.list}>
+            {appList.map((x, i) => (
+                <Fragment key={x.entry.ApplicationEntryID}>
+                    <AppListItem application={x} setUnlistedApp={setUnlistedApp} isListed={isListed} />
+                </Fragment>
```

Comment:
Looks like no need to wrap a single node with Fragment.

Reply:
list key.